app/account/forms.py

```diff
@@ -80,6 +80,16 @@ def validate_email(self, field):
             raise ValidationError('Unknown email address.')
 
 
+class CreatePasswordForm(Form):
+    password = PasswordField('Password', validators=[
```

Comment:
Agree with max here (https://www.owasp.org/index.php/Password_length_%26_complexity) suggests we make the minimum 8 characters. They also say that the recommendations may be out of date, but I think it's a good place to start. Also, we should probably add the validator to https://github.com/hack4impact/flask-base/blob/improved-user-flow/app/account/forms.py#L40 and https://github.com/hack4impact/flask-base/blob/improved-user-flow/app/account/forms.py#L70 and https://github.com/hack4impact/flask-base/blob/improved-user-flow/app/account/forms.py#L95.